devtools/build_requires.rst

```diff
@@ -1,6 +1,6 @@
 .. _build_requires:
 
-Build requirements
+Tool requirements
```

Comment:
I'm a bit concerned that users suddenly are not going to find anything about build requires because they have been aliased to tool_requires and we want them to use the new name thinking in 2.0, but I think that we should add a note in certain places like here explaining about the new name, and that now they should start naming their build_requires as tool_requires, if not this may be confusing... WDYT?

Reply:
I wouldn't like all users just replacing "build_requires" with "tool_requires" without really understanding that the scope is just for tools, and not for things like libraries (that they want to hide transitivity). I also see the other point, that if we are not pushing for this new naming, it won't happen because users will not realize.

Reply:
Well, the explanation of "this is only for tools" is everywhere, I'm not sure we can do much more. For me, if you read the documentation, you see "tool require" and you read "this is only for tools"..., this is an open-and-shut case.